docs/general/administration/hardware-acceleration/intel-hwa-tutorial.md

```diff
@@ -802,7 +754,7 @@ Root permission is required.
    - **Compute** - GPGPU or QSV VPP workload (only available on ARC / DG2+)
 
    ```shell
-   # intel_gpu_tool
+   $ sudo intel_gpu_top
```

Comment:
```suggestion
   sudo intel_gpu_top
```

Reply:
I'm actually going to be a bit contrarian here and say that I *prefer* having the "shell character" at the start. For one because it stops people blindly copying the commands without reading them, but also because it makes (or can make) clear what user(s) are expected to run the commands. See for instance my [rffmpeg documentation](https://github.com/joshuaboniface/rffmpeg/blob/master/SETUP.md).